bip-0039.mediawiki

```diff
@@ -146,6 +146,9 @@ Objective-C:
 Haskell:
 * https://github.com/haskoin/haskoin
 
+.NET (Standard):
+* https://www.nuget.org/packages/dotnetstandard-bip39/
```

Comment:
```suggestion
* https://github.com/elucidsoft/dotnetstandard-bip39
```

Instead if a link to the NuGet package (binary files, no source code), add a link to the GitHub repo where the implementation is.